styles/ui-variables.less

```diff
@@ -156,6 +156,7 @@
 
 // MISC
 @seti-font-family: 'Roboto', 'Helvetica Neue', 'Helvetica', 'Arial', sans-serif;
+@font-family: 'Roboto', 'Helvetica Neue', 'Helvetica', 'Arial', sans-serif;
```

Comment:
I think it would make more sense to set font family to setu-font-family

```
@font-family: @seti-font-family;
```

Reply:
I had the same error, and I made this change locally.